src/Diagnostics/BarPanel.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Contributte\RabbitMQ\Diagnostics;
+
+use Contributte\RabbitMQ\Producer\Producer;
+use Contributte\RabbitMQ\Producer\ProducerFactory;
+use Nette\Utils\Html;
+use Tracy\Helpers;
+use Tracy\IBarPanel;
+
+class BarPanel implements IBarPanel
+{
+
+	private ProducerFactory $producerFactory;
+
+	/**
+	 * @var array<string, string[]>
+	 */
+	private $sentMessages = [];
+
+	public function __construct(ProducerFactory $producerFactory)
+	{
+		$this->producerFactory = $producerFactory;
+
+		$this->producerFactory->onCreated[] = function (string $name, Producer $producer) {
+			$this->sentMessages[$name] = [];
+			$producer->onPublish[] = function (string $message) use ($name) {
+				$this->sentMessages[$name][] = $message;
```

Comment:
Optional: Could you maybe add a public static property with default value 100? That value would be checked in this anonymous function - when the count of stacked message reaches that value (100), other messages get trucated. In this case, it would be cool to show user that there were more then 100 messages and the rest got truncated. WDYT?

Reply:
sure, I'll add it